roles/pulp/tasks/install.yml

```diff
@@ -174,8 +174,7 @@
 
 - block:
 
-    # Hack for use system-wide packages when enabled by user - Must be first usage of virtualenv
-    - name: Allow use system-wide packages
+    - name: Create a venv with system-wide packages setting if the venv does not exist
       pip:
         name: pip
```

Comment:
Add a comment like "specifying pip is a safe hack to create the venv, since pip is automatically in every new venv."

Reply:
done!